src/main/java/io/vertx/ext/apex/core/impl/Utils.java

```diff
@@ -115,7 +116,7 @@ public static String readResourceToString(String resource) {
   }
 
   public static DateFormat createISODateTimeFormatter() {
-    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz");
+    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.US);
```

Comment:
The SimpleDateFormat object is created with locale setting of the system environment which means that parsing http date fields fails on systems with non-english locale since the parser expects localized day and month names, like e.g.
````Mi, 21 Jan 2015 20:43:10 MEZ
mer., 21 janv. 2015 20:43:10 CET
Wed, 21 Jan 2015 20:43:10 CET
on, 21 jan 2015 20:43:10 CET
````
(the first is german, the 2nd is french, the third is u.s. which is correct, the 4th is swedish)
usually the locale without any language specific setting is called "C" which turns localization completely off, but that is not used in Java, so US locale is correct I think (en_US, we could use UK "en_GB")